django/core/management/commands/makemessages.py

```diff
@@ -666,7 +648,7 @@ def process_locale_dir(self, locale_dir, files):
         else:
             return
 
-        input_files = [bf.work_path for bf in build_files]
+        input_files = [str(bf.work_path) for bf in build_files]
```

Comment:
This edit here seems to fix the windows failures.

```suggestion
        if os.name == "nt":
            input_files = [".\\" + str(bf.work_path) for bf in build_files]
        else:
            input_files = [str(bf.work_path) for bf in build_files]
```